### ISSUE TITLE
Review handling of iostat units change from sectors to kB between fields avgrq-sz and areq-sz

### ISSUE THREAD

**Issue by portante** [distributed-system-analysis/pbench#969]
See original discussion at: https://bugzilla.redhat.com/show_bug.cgi?id=1615896#c12

```
commit 772677945cbc9455d1c6edf7e04508c02be5fe7c
Author: Sebastien GODARD <sysstat@users.noreply.github.com>
Date:   Sun Jun 4 15:36:55 2017 +0200

    iostat: Express requests average size in kB, not sectors
    
    Since field "avgrq-sz" was renamed to "areq-sz", also change its unit:
    This is now a number of kilobytes, and not sectors.
    JSON output keeps both fields for backward compatibility: "avgrq-sz"
    expressed in sectors, and "areq-sz" expressed in kilobytes.
```

**Comment by ndokos** [comment]
Most of this was done in PR #1207, but there is one part of that PR that seems suspect to me: [line 117 in `iostat-postprocess`](https://github.com/distributed-system-analysis/pbench/blob/9e7cc4524e420820213c8da4f1f41f12b8d3229c/agent/tool-scripts/postprocess/iostat-postprocess#L117) defines `Request_Size_in_512_byte_sectors'  by adding the rareq-sz and wareq-sz values and multiplying the result by 2 - that seems wrong to me. If we had collected the areq-sz values  from iostat, then this field could be calculated by multiplying that by 2, but we don't.  @atheurer, can you please take a look?

**Comment by portante** [comment]
First there is a base units problem: 512 bytes (per sector) and 1,024 bytes (KB) fields.
 * If you have 100 KB, then you'd have 200 sectors
 * If you have 100 sectors, then you'd have 50 KB

Second there is a break down of read/write with `rareq-sz` and `wareq-sz` fields, where as the `areq-sz` field combines both.

So if you are going to convert from read + write KB into combined sectors, it is (rKB + wKB) * 2, since there are 2 sectors for every KB.

Isn't that what the current code is doing?

**Comment by ndokos** [comment]
Yes - I'm not questioning the multiplication by 2, I'm questioning the addition. If during a 10 second interval , I read 100 KB and I write 100KB then the avg req size for read is 10KB as is the avg req size for writes.So the avq request size for any operation is 10KB, not 20KB.
